.gitlab-ci.yml

```diff
@@ -595,14 +599,31 @@ changelog:
     - test -z "$RELEASE_PLEASE_PR" && echo "No release-please PR found" && exit 0
     - gh pr checkout --force $RELEASE_PLEASE_PR
     - wget --output-document cliff.toml https://raw.githubusercontent.com/mendersoftware/mendertesting/master/utils/cliff.toml
-    - mv CHANGELOG.md.${CI_COMMIT_REF_NAME} CHANGELOG.md
-    - git cliff
-      --unreleased
-      --prepend CHANGELOG.md
-      --github-repo ${GITHUB_REPO_URL}
-      --use-branch-tags
-      --tag $(jq -r '.["."]' .release-please-manifest.json)
-    - git add CHANGELOG.md
+    - if [[ "${THIS_REPO}" == "mender-server-enterprise" ]]; then
+        if [[ $(grep --count "saas" .release-please-manifest.json) -eq 1 ]]; then
+          echo "INFO - running CHANGELOG for saas only";
+          CHANGELOG_SUFFIX="-saas";
+          mv CHANGELOG${CHANGELOG_SUFFIX}.md.${CI_COMMIT_REF_NAME} CHANGELOG${CHANGELOG_SUFFIX}.md;
+          git cliff --unreleased --prepend CHANGELOG${CHANGELOG_SUFFIX}.md --github-repo ${GITHUB_REPO_URL} --use-branch-tags --tag $(jq -r '.["."]' .release-please-manifest.json);
```

Comment:
Can we reuse this `jq` expression for the version? I think it's better than using the grep expression of the entire manifest as this would conflict if we want to move other components to this repo.

Reply:
I think I don't got this.. Isn't the expression already using the `jq` command?

Reply:
I meant this line here: https://github.com/mendersoftware/mender-server/blob/7b5b82936bb30034d0ad203ab9c724f926868453/.gitlab-ci.yml#L603